workspaces/orchestrator/.changeset/popular-months-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@red-hat-developer-hub/backstage-plugin-orchestrator': patch
+---
+
+[UI] Layout errors with workflow details screen
```

Comment:
This message will be part of the changelog, one of many. I think this will not be clear to the reader without knowing wider context. A clear sentence will be better.